friends/src/components/UpdateFriend.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+class UpdateFriend extends React.Component {
+    state = {
+      item: this.props.activeItem
+    };
+  
+    changeHandler = ev => {
+      ev.persist();
+      let value = ev.target.value;
+  
+      this.setState(prevState => ({
+        item: {
+          ...prevState.item,
+          [ev.target.name]: value
```

Comment:
item was an id. Now it's an object with whatever we typed into the form, but it replaced the id.